Cod3rsGrowth.Forms/Program.cs

```diff
@@ -48,6 +46,20 @@ private static ServiceProvider CreateServices()
                     .AddScoped<IValidator<Carta>, CartaValidador>()
                     .AddScoped<IValidator<Baralho>, BaralhoValidador>()
                     .AddScoped<IValidator<Jogador>, JogadorValidador>()
+                    .AddScoped<Form1>();
+                });
+        }
+
+        private static ServiceProvider CreateServices()
+        {
+            string stringDeConexao = ConfigurationManager.ConnectionStrings[_stringDeConexao].ToString();
```

Comment:
não precisa desse to string, pode colocar só um ConnectionString no final

Reply:
Corrigi, continua funcionando